src/components/popover/popover.ts

```diff
@@ -47,9 +47,12 @@ export class Popover extends Component<Props, SpreadsheetChildEnv> {
     // within our control and to avoid leaking into external DOM
     const horizontalPosition = `left:${this.horizontalPosition() - this.spreadsheetPosition.x}`;
     const verticalPosition = `top:${this.verticalPosition() - this.spreadsheetPosition.y}`;
-    const height = `max-height:${
+    const max_height = Math.max(
+      0,
       this.viewportDimension.height - BOTTOMBAR_HEIGHT - SCROLLBAR_WIDTH
-    }`;
+    );
+    const height = `max-height:${max_height}`;
```

Comment:
Note for forwards port: in master/16.1, the commit 9d84b33fd260132bd09799455337e843bed43213 should have fixed all these issues regarding popovers.

Reply:
I already check this, and same issue is also in version 16.1 and in master. This is due to popover is not updating with resizing the screen.

Reply:
The position isn't updated, but the size should be. At least it is for me, maybe I'm missing something or its bugged. Could you show an example ?

Reply:
[https://www.awesomescreenshot.com/video/14439047?key=915c136cc7cffaacef55c82807c18aca](url)

Check this one.

Reply:
In your video : 

- indeed the font size dropdown bug, it isn't a Popover but your PR should fix the problem
- the problem with the popover is that we don't update its position when we resize the window. I'm pretty sure that if we did, the update in size would be automaticall handled. The behaviour in your video is "correct" for a Popover whose position is never updated : is stay at the same position no matter the window resize, and disappear if its anchor is outside de window.

You can probably create another task to fix the issue.